lib/pf9-builder/testcase-03-centos.csv

```diff
@@ -0,0 +1,10 @@
+## kubernetes
+# <hostname>|<image>:<username>|<flavor>|<network[,network,...]>|<key>|<security-group>|<role[,role]>[|<cluster-uuid>]
+k8s01|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s02|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s03|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s04|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s05|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s06|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s07|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
+k8s08|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af||cm-master|default|worker
```

Comment:
this should be pulled out, extra stuff a customer cannot use but we can share amongst ourselves